tasks/eslint.js

```diff
@@ -9,8 +9,15 @@ module.exports = grunt => {
 			quiet: false,
 			maxWarnings: -1,
 			failOnError: true,
+			extends: false
 		});
 
+		if (options.extends) {
+			options.baseConfig = Object.assign({
+				extends: options.extends
+			}, options.baseConfig);
```

Comment:
Should this option override any `extends` in `baseConfig`? If so, the order need to be reversed.

Reply:
The only common property between the two is the one I've added here. If somebody is using the baseConfig option, they're following ESlint's docs since it's not documented here. IMHO, it makes the most sense as is.